app/parallel_limiter.py

```diff
@@ -22,24 +21,16 @@ def __init__(
         lock_suffix: Optional[str] = None,
         max_wait_secs: int = 5,
```

Comment:
we can also remove `max_wait_secs` right?

Reply:
That defines what's the max time redis will hold the lock.